pkg/preparer/setup.go

```diff
@@ -255,10 +266,11 @@ func (c *PreparerConfig) getOpts() (consul.Options, error) {
 	}
 
 	return consul.Options{
-		Address: c.ConsulAddress,
-		HTTPS:   c.ConsulHttps,
-		Token:   token,
-		Client:  client,
+		Address:  c.ConsulAddress,
+		HTTPS:    c.ConsulHttps,
+		Token:    token,
+		Client:   client,
+		WaitTime: c.ConsulConfig.WatchWaitTime,
```

Comment:
since we weren't setting this on `consul.Options` before, it was already zero. It's defaulted to 5 minutes somewhere within consul